cve_bin_tool/checkers/expat.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/python3
 # pylint: disable=anomalous-backslash-in-string, invalid-name
-"""
+r"""
```

Comment:
I don't think we want a `r` prefix on this docstring

Reply:
There's an unescaped backslash in the docstring on [line 16](https://github.com/intel/cve-bin-tool/blob/24a95164831645a832e0d6daceff9b78b522d396/cve_bin_tool/checkers/expat.py#L16). Adding another backslash to escape it can change the meaning, which is why we can convert the docstring to a raw string.